service/policy/db/migrations/20240618000000_add_delete_cascade.sql

```diff
@@ -140,110 +140,122 @@ ON DELETE CASCADE;
 
 -- Do not cascade deletion of Attribute Definitions when their parent Namespace is deleted
 ALTER TABLE attribute_definitions
-DROP CONSTRAINT attribute_definitions_namespace_id_fkey_cascades;
-
+DROP CONSTRAINT IF EXISTS attribute_definitions_namespace_id_fkey_cascades;
+ALTER TABLE attribute_definitions
+DROP CONSTRAINT IF EXISTS attribute_definitions_namespace_id_fkey; -- Add this
```

Comment:
Could you please remove the `-- Add this` comments throughout this file for clarity?